proto/forch_configuration.proto

```diff
@@ -150,6 +150,9 @@ message OrchestrationConfig {
 
     // DUT will return to operational state if no DeviceReport is received within this timeout.
     int32 sequester_timeout_sec = 6;
+
+    // Default value for DeviceBehavior.auto_sequestering
+    bool default_auto_sequestering = 7;
```

Comment:
Should be of the same type as the behavior, I think... so PortBehavior.AutoSequestering so it's very clear how it matches.

Reply:
But that means you can have a “default” as the value of default. What’s the default value for the “default” of default?